### DISCUSSION TITLE
Migrate Confluence Docs into Github

### DISCUSSION THREAD

**Discussion by arolfes** [Taskana/taskana#2117] [Ideas]
Hallo @holgerhagen , @gitgoodjhe , @mustaphazorgati ,

did you discuss a migration of the confluence docs into Github Wiki or even better into AsciiDoc or MarkDown

**Comment by mustaphazorgati** [comment]
We did. We agreed on migrating Confluence aswell when we we want to migrate JIRA -> GitHub.
This could be a very good opportunity to update and maybe restructure our existing documentation, so that we have a "clean" documentation here on GitHub

**Reply by mustaphazorgati** [reply]
GitHubs Wiki (https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis) seems to be a very nice option. Unfortunately it lacks one functionality: Versioning. 
I've read a small guide somewhere where you can introduce versioning into GitHub Wikis. But I didn't find it yet.

**Comment by arolfes** [comment]
What do you think of markdown files or AsciiDoc files for the documentation and then publish it to readthedocs hoster...

**Reply by mustaphazorgati** [reply]
I personally prefer Markdown files, but we already use AsciiDoc for our REST documentation. To be consistent we could use AsciiDoc
